etcdserver/errors.go

```diff
@@ -39,7 +39,7 @@ var (
 	ErrKeyNotFound                = errors.New("etcdserver: key not found")
 	ErrCorrupt                    = errors.New("etcdserver: corrupt cluster")
 	ErrBadLeaderTransferee        = errors.New("etcdserver: bad leader transferee")
-	ErrMemberRemoved              = errors.New("etcdserver: the member has been permanently removed from the cluster")
+	ErrMemberRemoved              = errors.New("the member has been permanently removed from the cluster")
```

Comment:
Why did this get backed out?

Reply:
okay so this is weird behavior i noticed, when the error is sent to the channel, its only the "the member has been...." not etcdserver:, and I think this is for the zap logger to recognize it as a server error, so when doing that I cant really compare the error that is being sent to the channel with this constant, thats why i use only plain error message without a etcdserver header

Reply:
I know there's some weirdness with error types coming over grpc - can you link to where we're checking the error that's not comparing correctly?